Source/src/Batching/TextureBatch.cpp

```diff
@@ -247,12 +249,12 @@ void Hachiko::TextureBatch::UpdateBatch(int segment, const std::vector<const Com
 void Hachiko::TextureBatch::BindBatch(int segment, const Program* program, unsigned component_count)
 {
     // Bind textures
-    const std::vector<int> texture_slots = {10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26};
+    const std::vector<int> texture_slots = {11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26};
     program->BindUniformInts("allMyTextures", texture_arrays.size(), &texture_slots[0]);
 
     for (unsigned i = 0; i < texture_arrays.size(); ++i)
     {
-        glActiveTexture(GL_TEXTURE10 + i);
+        glActiveTexture(GL_TEXTURE11 + i);
```

Comment:
I believe these were changed because of dissolve? It may be a good idea to review these after re-assigning the binding position of noise on the shader, right now, binding 25 is used by the batches for example.

Reply:
The binding is not related to the texture slot, the binding is for avoiding the translation we have to make each time we want to get a uniform by name so you can use that. Now the texture slot of the noise is the number 10 (the closer one to the used ones) so i had to shift those one more.